### DISCUSSION TITLE
osu!cookie size

### DISCUSSION THREAD

**Discussion by RaoraRare** [ppy/osu#24873] [Ideas] [answered]
I think the osu!cookie should be resizeable without UI/UX, like there should be a separate bar thing to change the size.

Some people like me might like to have small UI/UX and a big cookie, I'm not saying it's a good feature but it would allow for more customization.

Example:
![Untitled1_20230920091359.png](https://github.com/ppy/osu/assets/120297366/b0f29bda-5e9f-46cb-83cd-570b5dddb8ff)

(Edit If so I could try to add it myself)

**Comment by peppy** [comment, marked as answer]
Not going to happen sorry.

**Reply by RaoraRare** [reply]
Ty for the answer
